flink-connectors/flink-hbase/pom.xml

```diff
@@ -34,7 +34,7 @@ under the License.
 	<packaging>jar</packaging>
 
 	<properties>
-		<hbase.version>1.3.1</hbase.version>
+		<hbase.version>1.4.0</hbase.version>
```

Comment:
1.4.1?

Reply:
Thanks @greghogan It is a typo. Will fix.